flink-runtime/src/test/java/org/apache/flink/runtime/io/network/api/writer/RecordWriterTest.java

```diff
@@ -510,6 +512,14 @@ public void flushAll() {
 		public void flush(int subpartitionIndex) {
 		}
 
+		@Override
+		public void fail(@Nullable Throwable throwable) {
```

Comment:
Why not keep the same way of `UnsupportedOperationException` as others?

Reply:
True, I missed it, thanks :)